creating_images/guidelines.adoc

```diff
@@ -138,34 +133,29 @@ For example, if you are working on a *_Dockerfile_* that contains an `ADD`
 command to install a file you are iterating on, and a `RUN` command to `yum
 install` a package, it is best to put the `ADD` command last:
 
-====
 
 ----
 FROM foo
 RUN yum -y install mypackage && yum clean all -y
 ADD myfile /test/myfile
 ----
-====
-
+ifndef::openshift-online[]
 This way each time you edit *_myfile_* and rerun `docker build`, the system reuses
 the cached layer for the `yum` command and only generates the new layer for the
 `ADD` operation.
 
 If instead you wrote the *_Dockerfile_* as:
 
-====
-
 ----
 FROM foo
 ADD myfile /test/myfile
 RUN yum -y install mypackage && yum clean all -y
 ----
-====
 
 Then each time you changed *_myfile_* and reran `docker build`, the `ADD`
 operation would invalidate the `RUN` layer cache, so the `yum` operation would
 need to be rerun as well.
-
+endif::[]
```

Comment:
Same with this. This describes running the `docker build` command outside of OpenShift.